django_loci/base/models.py

```diff
@@ -85,7 +85,7 @@ def short_type(self):
 
 class AbstractFloorPlan(TimeStampedEditableModel):
     location = models.ForeignKey('django_loci.Location', on_delete=models.CASCADE)
-    floor = models.SmallIntegerField(_('floor'))
+    floor = models.SmallIntegerField(_('floor'), null=False, blank=False)
```

Comment:
this should not be necessary because these are the default values for fields in django, see:

- https://docs.djangoproject.com/en/3.1/ref/models/fields/#null
- https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.Field.blank

Reply:
Oh, I see. Thanks for letting me know! Going to remove that next time :smile: